include/egg/math/eggQuat.h

```diff
@@ -0,0 +1,47 @@
+#include <egg/math/eggVector.h>
+
+namespace EGG {
+
+struct Quatf {
+    Vector3f vec;
```

Comment:
Does this work instead?
```suggestion
    union {
        struct {
            f32 x, y, z;
        };
        Vector3f v;
    };
```
Optionally, you can also move the members of the struct below the methods.